src/common/data/stores/app/space/spaceStore.ts

```diff
@@ -129,6 +129,19 @@ export const createSpaceStoreFunc = (
       const spaceConfig = JSON.parse(
         await get().account.decryptEncryptedSignedFile(fileData),
       ) as DatabaseWritableSpaceConfig;
+      const currentLocalCopy = get().space.localSpaces[spaceId];
+      if (
+        spaceConfig &&
+        spaceConfig.timestamp &&
```

Comment:
Should also return true if local config has a timestamp and the fetched one doesn't